src/Microsoft.Net.Sdk.AnalyzerRedirecting/Microsoft.Net.Sdk.AnalyzerRedirecting.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <Nullable>enable</Nullable>
+
+    <IsShipping>false</IsShipping>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+
+    <!-- VSIX -->
+    <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
+    <GeneratePkgDefFile>true</GeneratePkgDefFile>
+    <IncludeAssemblyInVSIXContainer>true</IncludeAssemblyInVSIXContainer>
+    <IncludeDebugSymbolsInVSIXContainer>true</IncludeDebugSymbolsInVSIXContainer>
+    <IncludeDebugSymbolsInLocalVSIXDeployment>true</IncludeDebugSymbolsInLocalVSIXDeployment>
+    <IncludeCopyLocalReferencesInVSIXContainer>false</IncludeCopyLocalReferencesInVSIXContainer>
+    <VsixVersion Condition="'$(VsixVersion)' == ''">42.42.42.4242424</VsixVersion>
```

Comment:
when is this ever set to a different value?

Reply:
Hm, good point, I thought arcade would set this, but maybe something is missing.

Reply:
It does look like the arcade targets set this. src/Microsoft.DotNet.Arcade.Sdk/tools/VisualStudio.targets. I think it does set this to a default version if not specified. So maybe this line is not necessary?

Reply:
Ah, okay, I've remembered why this is needed. (I will add a comment.)

Arcade [imports VisualStudio.targets](https://github.com/dotnet/arcade/blob/00bb044268c007ed27c5ab50c5839fcf36fdff0f/src/Microsoft.DotNet.Arcade.Sdk/tools/Imports.targets#L18) only on .NET Framework MSBuild. But SDK's CI builds are using Core MSBuild. Filed also https://github.com/dotnet/arcade/issues/15617.